source/MaterialXGenShader/ShaderNode.cpp

```diff
@@ -292,14 +290,6 @@ ShaderNodePtr ShaderNode::create(const ShaderGraph* parent, const string& name,
     {
         newNode->_classification = Classification::TEXTURE | Classification::DOT;
     }
-    else if (nodeDef.getNodeString() == COMPARE)
-    {
-        newNode->_classification = Classification::TEXTURE | Classification::CONDITIONAL | Classification::IFELSE;
-    }
-    else if (nodeDef.getNodeString() == SWITCH)
```

Comment:
I'm not sure way it means to remove the texture classification here. I think nodes like try to perform filtered sample upstream will no longer work. But it could be that it does not work anyways.